scripts/test.sh

```diff
@@ -69,7 +69,7 @@ cleanup() {
   pkill -9 -P $$ || true
 
   # Kill react native packager
-  pkill node || true
+  pkill -x node || true
```

Comment:
This isn't related to the rest of the change, but it fixes warnings that are printed about trying and failing to kill root-owned `ionodecache` processes.